config/foreman-answers.yaml

```diff
@@ -9,8 +9,7 @@
 #
 # See params.pp in each class for what options are available
 ---
-foreman:
-  configure_epel_repo: false
+foreman: {}
```

Comment:
Note I chose a hash since our migrations aren't written cleanly. `true` would be cleaner, but this matches `foreman_proxy` below.